src/components/FloatingDropdown/FloatingDropdown.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+
+import { Dropdown as CoreDropdown } from 'wix-ui-core/dropdown';
+
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import styles from './FloatingDropdown.st.css';
+import { DropdownOption } from '../Dropdown/DropdownOption';
```

Comment:
@jonathanadler same with Options styling and props, use what we already have locally, OK?